pkg/apis/externaldns/types_test.go

```diff
@@ -176,7 +176,7 @@ var (
 		AzureSubscriptionID:         "arg",
 		CloudflareProxied:           true,
 		CloudflareDNSRecordsPerPage: 5000,
-		CloudflareRegionKey: 		 "us",
+		CloudflareRegionKey:         "us",
```

Comment:
Please avoid unrelated changes. Consider filing a separate PR for linting instead